docs/source/guides/user/chapters/introduction.rst

```diff
@@ -315,55 +315,64 @@ output in the standard output of the runner, simply use::
 
 **2. JSON:**
 
+.. warning:: The result.json test attributes related to time has been renamed in version 104
```

Comment:
I believe this is material for the release notes.  By adding version specific documentation to the users guide, we make it hard for users to find the changes that happened between versions.